Packs/IronPort/Integrations/CiscoEmailSecurityApplianceIronPortV2/CiscoEmailSecurityApplianceIronPortV2.yml

```diff
@@ -561,188 +561,218 @@ script:
       contextPath: CiscoESA.Message.sbrs
       description: Sender Base Reputation Scores.
   - name: cisco-esa-message-details-get
-    description: Get message details.
+    description: Retrieve metadata and tracking information for email messages processed by the ESA. Includes sender/recipient details, timestamps, message status (e.g., delivered, dropped), message size, AMP and DLP visibility flags, and applied mail policies — useful for auditing and threat tracing.
     arguments:
-    - name: serial_number
-      description: Email gateway serial number.
-      required: true
-    - name: message_ids
-      description: Message ID list.
-      required: true
-      isArray: true
-    - name: injection_connection_id
-      description: Injection connection ID.
+      - name: injection_connection_id
+        description: An Injection Connection ID (ICID). A numerical identifier for an individual SMTP connection to the system. Use `cisco-esa-message-search` to get message Injection Connection IDs. Consider to insert this parameter in case you don't achive any results.
```

Comment:
An Injection Connection ID (ICID). A unique numerical ID for a single SMTP connection. Use 'cisco-esa-message-search' to get message ICIDs. Recommended if no results are found without it.